scripts/pre_commit_linter.py

```diff
@@ -100,14 +100,14 @@
 
 BAD_PATTERNS_JS_REGEXP = [
     {
-        'regexp': r"\b(ddescribe|fdescribe)\(",
+        'regexp': r"\b(ddescribe|fdescribe)\(", # pylint: disable=hanging-indent
```

Comment:
Could this be a result of mixed tabs and spaces? (Looking at [this](https://stackoverflow.com/questions/25303726/how-to-fix-pylint-wrong-hanging-indentation-and-pep8-e121) stackoverflow)

Reply:
Hi @kevinlee12, the custom check which I have added checks if there is a comma-separated list within braces when the opening and closing braces are not on the same line. In this case, since there is a opening bracket and then some content (which is double quotes) and then a comma, the check marks this as an error, so I have disabled the check here. Should I add some checking which checks if the content is not double quotes to avoid disabling the check?

Reply:
Having that condition would be preferable, since we should disable checks as little as possible.

Reply:
Done @kevinlee12, PTAL!